daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Visual.hs

```diff
@@ -50,6 +50,7 @@ startFromExpr seen world e = case e of
     LF.ERecProj _ _ recEx -> startFromExpr seen world recEx
     LF.ETupleUpd _ recExpr recUpdate -> startFromExpr seen world recExpr `Set.union` startFromExpr seen world recUpdate
     LF.EVariantCon _ _ varg -> startFromExpr seen world varg
```

Comment:
Generics required!

Reply:
I guess that's @akalburgi's business. ;)

Reply:
Agreed! More a remark for him that the lack of generics is starting to have a cost on other changes.

Reply:
Currently working on it

Reply:
@hurryabit using uniplate PR https://github.com/digital-asset/daml/pull/1826